lib/widgets/scrollable_screen_container.dart

```diff
@@ -4,12 +4,14 @@ class ScrollableScreenContainer extends StatelessWidget {
   final ScrollController? scrollController;
   final EdgeInsetsGeometry? padding;
   final Widget child;
+  final ScrollPhysics? physics;
 
   const ScrollableScreenContainer({
     super.key,
     this.scrollController,
     this.padding,
     required this.child,
+    this.physics,
```

Comment:
I think that it would be a good idea to have this `physics` named property with the default value of `ClampingScrollPhysics()`, this way, we don't need to go and change everywhere we use it